src/App.css

```diff
@@ -20,6 +20,12 @@
 /* --------------- Testimonial Video's ------------------------- */
 .testimonial-section {
     background-color: var(--color-background-light);
+    display: flex;
+    padding: 1rem;
```

Comment:
Padding op de boven- en onderkant is prima, maar in dit geval niet aan de zijkanten omdat de container dan niet meer uitlijnt met de rest van de pagina!